anonlink/blocking.py

```diff
@@ -0,0 +1,236 @@
+from abc import ABCMeta, abstractmethod
+from itertools import chain, product, repeat
+from numbers import Real
+from random import Random
+from typing import (Any, Callable, Hashable, Iterable, List,
+                    Optional, overload, Sequence, Tuple, TypeVar, Union)
+
+
+BlockingFunction = Callable[[int, int, Sequence[bool]], Iterable[Hashable]]
+
+
+_T = TypeVar('_T')
+def _evalf(__funcs : 'Iterable[Callable[..., _T]]',  # https://github.com/python/typing/issues/259
+           *args,
+           **kwargs) -> Iterable[_T]:
+    """ Apply the same arguments to a number of functions.
```

Comment:
"Apply a number of functions to the same arguments."

Reply:
Thanks, that does make more sense